### PR TITLE
Forbid mapping the I drive

### PR DESCRIPTION
**Pull request by weshinsley**

### CODE REVIEW

File: drivers/windows/R/paths.R [hunk 1]
```diff
@@ -5,8 +5,7 @@ windows_path <- function(name, path_local, path_remote, drive_remote) {
   assert_scalar_character(drive_remote)
 
   if (!grepl("^[A-Za-z]:$", drive_remote)) {
-    stop(sprintf("drive_remote must be of the form 'X:' (but was '%s')",
-                 drive_remote))
+    cli::cli_abort("drive_remote must be of the form 'X:' (but was '%s')")
```
Comment (richfitz) [inline drivers/windows/R/paths.R:8]:
```suggestion
    cli::cli_abort("drive_remote must be of the form 'X:' (but was '{drive_remote}')")
```

File: drivers/windows/R/paths.R [hunk 2]
```diff
@@ -15,11 +14,20 @@ windows_path <- function(name, path_local, path_remote, drive_remote) {
...
 
   if (!file.exists(path_local)) {
-    stop("Local mount point does not exist: ", path_local)
+    cli::cli_abort(c("Local mount point does not exist.",
+                 i = "Please check the mount point: {path_local}."))
+  }
+
+  if (drive_remote == "I:") {
+    cli::cli_abort(c("You cannot use I: on a cluster job.",
+                 i = "I: is reserved for package provisioning.",
```
Comment (richfitz) [inline drivers/windows/R/paths.R:29]:
```suggestion
                 i = "I: is reserved for internal use.",
```

More mysterious :shipit:

**Review by richfitz — CHANGES_REQUESTED** [review]

File: drivers/windows/R/paths.R (hunk 1 above)
Comment (weshinsley) [reply drivers/windows/R/paths.R:8]: Thanks - how did I miss that...!

File: drivers/windows/R/paths.R (hunk 2 above)
Comment (weshinsley) [reply drivers/windows/R/paths.R:29]: hipercow moves in mysterious ways, its wonders to perform.

**Review by richfitz — APPROVED** [review]

**Pull request merged**